website/blog/2020-04-10-cobalt.md

```diff
@@ -0,0 +1,350 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.4 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are happy to announce the release of Metals v0.8.4, which will most likely be
+the last 0.8.x version.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">174</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">55</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">9</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">13</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">3</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/22?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- navigating to parent method
+- support for `launch.json`
+- new "Create new symbol" code action
+- miscellaneous improvements
+
+## Navigating to parent method
+
+Metals will now calculate if any definition of a method in the current file
+overrides another method, in which case it will show new lens next to the
+method's position. Once clicked, the user will be able to go to the super
+method. Lenses also link to methods in external dependencies and are displayed
+in files from libraries.
+
+![Screenshot 2020-03-10 at 09 24 41](https://user-images.githubusercontent.com/10478402/76293237-fad12c80-62b0-11ea-9952-d8b536ffd374.png)
+
+If they are found to be disturbing your workflow they can be easily disabled via
+the `metals.superMethodLensesEnabled` configuration option - they are enabled by
+default. For example in Visual Studio Code:
+
```

Comment:
Is it maybe also worth putting in a reminder that in order for these the work, ensure you have code lenses enabled in your client? I'm not sure if they are for vscode, but for example they aren't be default in coc.nvim.

Reply:
Added a mention, good idea!